### PR TITLE
Added tests for the Review page

### PR DESCRIPTION
**Pull request by sagar-1310**
## Proposed changes

This PR addresses Issue: [*Link to Github issue within https://github.com/zowe/zen/issues* if any]

This PR depends upon the following PRs:

## Type of change
Please delete options that are not relevant.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Change in a documentation
- [ ] Refactor the code 
- [ ] Chore, repository cleanup, updates the dependencies.
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## PR Checklist
Please delete options that are not relevant.
- [ ] If the changes in this PR are meant for the next release / mainline, this PR targets a "staging" branch.
- [ ] My code follows the style guidelines of this project (see: [Contributing guideline](https://github.com/zowe/zen/blob/v2.x/staging/CONTRIBUTING.md))
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] New and existing unit tests pass locally with my changes
- [ ] video or image is included if visual changes are made

- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works, or describe a test method below

### CODE REVIEW

File: playwright_test/Tests/Review.spec.ts [hunk 1]
```diff
@@ -0,0 +1,254 @@
...
+    expect(reviewPage.viewEditYaml).toBeTruthy()
+    expect(reviewPage.viewSubmitJob).toBeTruthy()
+    expect(reviewPage.viewJobOutput).toBeTruthy()
+    expect(reviewPage.saveAndClose).toBeTruthy()
+    expect(reviewPage.previousStep).toBeTruthy()
+    const is_Finish_Button_disable = await reviewPage.isFinishInstallationDisabled();
+    expect(is_Finish_Button_disable).toBe(true);
+	})
+
+  test('Test Navigation to all pages', async ({ page }) => {
```
Comment (Himani1519) [inline playwright_test/Tests/Review.spec.ts]: Instead doing Navigation test in one test function, keep them in separate function

File: playwright_test/Tests/Review.spec.ts [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+    connectionPage.fillConnectionDetails(SSH_HOST,SSH_PORT,SSH_USER,SSH_PASSWD)
+    connectionPage.SubmitValidateCredential()
+    await page.waitForTimeout(5000);
+    connectionPage.clickContinueButton()
+    await page.waitForTimeout(2000);
+    planningPage.clickSaveValidate()
+    await page.waitForTimeout(20000);
+    planningPage.fillPlanningPageWithRequiredFields(RUNTIME_DIR, ZOWE_WORKSPACE_DIR,ZOWE_EXTENSION_DIR,ZOWE_LOG_DIR,'1',JOB_NAME,JOB_PREFIX,JAVA_HOME,NODE_HOME,ZOSMF_HOST,ZOSMF_PORT,ZOSMF_APP_ID)
+    await page.waitForTimeout(20000);
+    planningPage.clickValidateLocations()
```
Comment (Himani1519) [inline playwright_test/Tests/Review.spec.ts]: You can also directly move to Review tab, instead going through skip button.

File: playwright_test/Tests/Review.spec.ts [hunk 3]
```diff
@@ -0,0 +1,254 @@
+import { test, ElectronApplication, expect, _electron as electron } from '@playwright/test';
```
Comment (Himani1519) [inline playwright_test/Tests/Review.spec.ts:1]: Also You need to add a test case where state of each page should be match with the state of Review page.

**Comment by sagar-1310** [comment]
New PR: https://github.com/zowe/zen/pull/264

**Pull request closed without merging**